_includes/footer/desktop.html

```diff
@@ -24,7 +24,7 @@
     </ul>
   </div>
   <div class="footer-links">
-    <h5 class="footer-links-heading">Policies and Standards</h5>
+    <h4 class="footer-links-heading">Policies and Standards</h4>
```

Comment:
This is what @nickbristow recommended when I asked him. I think that `h4` is continuing the heading level from the previous section, but since the footer starts a new section, so it's OK to reset it.